src/site/content/en/metrics/inp/index.md

```diff
@@ -101,7 +99,7 @@ To ensure you're delivering user experiences with good responsiveness, a good th
 Since INP is experimental, the guidance around thresholds may change over time as it is fine-tuned. [The CHANGELOG at the end of this article](#changelog) will be updated to reflect any changes.
 {% endAside %}
 
-## What's in an interaction?
+### What's in an interaction?
```

Comment:
Why are these headings being brought in one more level?

Reply:
See above screenshots in first comment.

The other metrics all have this H2 outline, but INP is the exception - hence this PR:

- What is XXX?
- XXX in details (optional - it's there for CLS and FID, though LCP does not have this, so I skipped it here too to make the change easier rather than having to rearrange the text or add a new paragraph to avoid a stacked heading).
- How to measure
- How to improve
- Additional resources (optional - it's there for CLS and LCP, though FID does not have this so skipped it for now but do think we'll want to add it later)
- CHANGELOG